app/models/user/owned.rb

```diff
@@ -0,0 +1,21 @@
+module User
+  class Owned
+    attr_reader :user
+
+    def initialize(user)
+      @user = user
+    end
+
+    def uid
+      user.uid
+    end
+
+    def with_cache &block
```

Comment:
@developish - What is the purpose of this method? Is this just a placeholder that gets overridden in sub-classes? I don't see any other `with_cache` methods defined in this PR that override this method.

Reply:
This class was from an idea I started to work on, then backed out of